spec/models/reviewable_score_spec.rb

```diff
@@ -112,45 +112,45 @@
     it "returns the users weighted accuracy bonus" do
       user_stat.flags_agreed = 10
       user_stat.flags_disagreed = 42
-      expect(ReviewableScore.user_accuracy_bonus(user).floor(2)).to eq(-10.34)
+      expect(ReviewableScore.user_accuracy_bonus(user)).to be_within(0.1).of(-10.34)
```

Comment:
Sorry for a lack of context, why do we need this change and why is it variable now?

Reply:
Tests were flaky due to a chain of truncation of floats (round -> floor).

Reply:
:+1: Thank you for clarifying